todos/views.py

```diff
@@ -11,9 +14,15 @@ def get_queryset(self):
         """Return all the latest todos."""
         return Todo.objects.order_by('-created_at')
 
+
 def add(request):
     title = request.POST['title']
-    Todo.objects.create(title=title)
+    deadline = request.POST['deadline']
+    deadline_date = datetime.datetime.strptime(deadline, '%Y-%m-%d')
```

Comment:
We are getting an error that unconverted data remains: T05:23 for the line     deadline_date = datetime.datetime.strptime(deadline, '%Y-%m-%d')